src/main/java/uk/gov/hmcts/darts/arm/service/impl/ArmResponseFilesProcessorImpl.java

```diff
@@ -100,6 +100,11 @@ public void processResponseFiles() {
         }
     }
 
+    @Transactional
```

Comment:
I was of the understanding that `@Transactional` has no effect on private methods. Are you sure this fixes the issue?

Reply:
Well spotted